### DISCUSSION TITLE
Using different "g" glyph

### DISCUSSION THREAD

**Discussion by Akselmo** [IBM/plex#464] [Q&A] [answered]
Hi, first of all, i love the font

Second, I would like to set this font as my system font, but I would also like to switch the `g` glyph to the one i've underlined in the image.
![image](https://user-images.githubusercontent.com/13801236/184194636-133b335a-4c0c-4291-97e3-0792b9528b87.png)

I know nothing about fonts so if anyone can point me in the right direction I would be grateful.

I use Fedora Linux (KDE).

**Comment by akulisch** [comment, marked as answer]
This might help you: https://wiki.archlinux.org/title/Font_configuration/Examples#Alternate_stylistic_sets_for_fonts
for the single-storey g you need ss02

**Reply by Akselmo** [reply]
Can't get this to work. I made this file: `~/.config/fontconfig/fonts.conf`

```xml
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
<fontconfig>
    <match target='font'>
        <test name='fontformat' compare='not_eq'>
            <string/>
        </test>
        <test name='family'>
            <string>IBM Plex Sans</string>
        </test>
        <edit name='fontfeatures' mode='assign_replace'>
            <string>ss02</string>
        </edit>
    </match>
</fontconfig>
```
Tried to logout and log back in

If it affects anything, I'm using Wayland

**Reply by Akselmo** [reply]
Okay, this seems to be an issue with KDE not following font configs. GTK apps changed the glyph with this.

Thanks for the help! I'll ask KDE folks for help and will report here as well.

**Reply by Akselmo** [reply]
Seems I'm not alone with the issue: https://bugs.kde.org/show_bug.cgi?id=407055

**Reply by Akselmo** [reply]
I found the answer long time later :) With this script I could change all the font glyphs to the other g: https://codeberg.org/akselmo/script-things/src/branch/main/change_g.sh
